website/templates/profile/notifications.mako

```diff
@@ -65,7 +74,9 @@
     <% import website %>
     ${parent.javascript()}
     <script type="text/javascript">
-        window.contextVars = $.extend({}, window.contextVars, {'mailingList': ${website.settings.MAILCHIMP_GENERAL_LIST | sjson, n }});
+        window.contextVars = $.extend({}, window.contextVars, {
+            'mailingList': [${website.settings.MAILCHIMP_GENERAL_LIST | sjson, n }, ${website.settings.OSF_GENERAL_LIST | sjson, n }],
```

Comment:
This should say `mailingLists`. Also, please remove the unnecessary comma after the closing bracket.